### PR TITLE
Hotfix/no images

### PR DESCRIPTION
**Pull request by abea**
The request URL is now only returning minimal information unless more is specified. This adds the fields to the request that are used in the module. I also edited the treatment of photo posts to insert `alt` tag information.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -126,7 +127,9 @@ function Construct(options, callback) {
 
     //fb.setAccessToken(access_token);
```
Comment (plantainrain) [inline index.js]: 💣  if not needed

File: index.js [hunk 2]
```diff
@@ -76,7 +76,8 @@ function Construct(options, callback) {
   app.get('/apos-facebook/photo', function(req, res){
     //Grab the post ID and build out a request URL.
     var postId = req.query.id,
-        requestUrl = 'https://graph.facebook.com/'+postId+'?access_token='+access_token;
+        requestUrl = 'https://graph.facebook.com/' + postId + '?access_token=' +
+        access_token + '&fields=source,name,created_time';
```
Comment (plantainrain) [inline index.js]: break out into separate lines each with own 'var' for readability

**Review by plantainrain — CHANGES_REQUESTED** [review]

**Comment by abea** [comment]
And updated. I almost went in on a full code cleanup, but that was a deep rabbit hole.

**Review by bobclewell — APPROVED** [review]
lgtm

**Pull request merged**